torch/distributions/gamma.py

```diff
@@ -54,3 +55,12 @@ def log_prob(self, value):
     def entropy(self):
         return (self.alpha - torch.log(self.beta) + torch.lgamma(self.alpha) +
                 (1.0 - self.alpha) * torch.digamma(self.alpha))
+
+
+@register_kl(Gamma, Gamma)
+def _kl_gamma_gamma(p, q):
```

Comment:
This is only intended to demonstrate usage. @vishwakftw may implement a more efficient version of this.

Reply:
Maybe this should be public so users aren't force to use PyTorch's `kl_divergence()` registration mechanism?